src/App.tsx

```diff
@@ -1,12 +1,21 @@
 import { Route, Routes } from "react-router-dom";
 import { Home } from "./components/Home";
 import { Layout } from "./components/Layout";
+import { Task1 } from "./components/Task1";
+import { FileList } from "./components/FileList";
+import { FileItem } from "./components/FileItem";
+import { CreateFileForm } from "./components/CreateFileForm";
 
 const App = () => {
   return (
     <Routes>
       <Route path="/" element={<Layout />}>
         <Route path="/" element={<Home />} />
+        <Route path="/task1" element={<Task1 />}>
+          <Route index element={<FileList />} />
+          <Route path=":filename" element={<FileItem />} />
```

Comment:
I think leading slash should be added here

Reply:
If i do that, i get this
![image](https://user-images.githubusercontent.com/68477535/184618875-2d4464ae-0b23-4fe2-84a0-8400465936f3.png)

Reply:
I see, actually this was valid in router 5. Looks like a breaking change since then
Ok, leave it as is